08-transport.md

```diff
@@ -378,129 +313,86 @@ for the `poly1305` tag.
    -> s, se
 ```
 
-
-`Act Three` is the final phase in the authenticated key agreement described in
-this section. This act is sent from the initiator to the responder as a final
-concluding step. `Act Three` is only executed `iff` `Act Two` was successful.
-During `Act Three`, the initiator transports its static public key to the
-responder encrypted with _strong_ forward secrecy using the accumulated `HKDF`
+Act Three is the final phase in the authenticated key agreement described in
+this section. This act is sent from the initiator to the responder as a
+concluding step. Act Three is executed _if and only if_ Act Two was successful.
+During Act Three, the initiator transports its static public key to the
+responder encrypted with _strong_ forward secrecy, using the accumulated `HKDF`
 derived secret key at this point of the handshake.
 
-
-The handshake is _exactly_ `66 bytes`: `1 byte` for the handshake version, `33
-bytes` for the ephemeral public key encrypted with the `ChaCha20` stream
-cipher, `16 bytes` for the encrypted public key's tag generated via the `AEAD`
-construction, and `16 bytes` for a final authenticating tag.
-
+The handshake is _exactly_ 66 bytes: 1 byte for the handshake version, 33
+bytes for the ephemeral public key encrypted with the `ChaCha20` stream
+cipher, 16 bytes for the encrypted public key's tag generated via the AEAD
+construction, and 16 bytes for a final authenticating tag.
 
 **Sender Actions:**
 
-
   * `c = encryptWithAD(temp_k2, 1, h, s.pub.serializeCompressed())`
-    * where `s` is the static public key of the initiator.
-
-
+    * where `s` is the static public key of the initiator
   * `h = SHA-256(h || c)`
-
-
   * `ss = ECDH(re, s.priv)`
     * where `re` is the ephemeral public key of the responder.
-
-
   * `ck, temp_k3 = HKDF(ck, ss)`
-    * Mix the final intermediate shared secret into the running chaining key.
-
-
+    * The final intermediate shared secret is mixed into the running chaining key.
   * `t = encryptWithAD(temp_k3, 0, h, zero)`
      * where `zero` is a zero-length plaintext
-
-
   * `sk, rk = HKDF(ck, zero)`
      * where `zero` is a zero-length plaintext,
-
-
        `sk` is the key to be used by the initiator to encrypt messages to the
        responder,
-
-
        and `rk` is the key to be used by the initiator to decrypt messages sent by
-       the responder.
-
-     * This step generates the final encryption keys to be used for sending and
-       receiving messages for the duration of the session.
-
+       the responder
+     * The final encryption keys to be used for sending and
+       receiving messages for the duration of the session are generated.
```

Comment:
the final...generated

Reply:
It's a complete sentence, not in a requirements list, so it should be capitalized and have a period, yes?

Reply:
Oops, sorry you're right.

Reply:
Great! I was momentarily confused!